packages/cli/src/models/config/ConfigManager.ts

```diff
@@ -78,6 +78,11 @@ const ConfigManager = {
     return config && config.networks ? Object.keys(config.networks) : undefined;
   },
 
+  getConfigFileName(root: string = process.cwd()): string {
```

Comment:
It seems like this new method is only used for the nice error message? I'd expect for this PR to also use it when reading from the network file.

Reply:
@nventuro fixed in `NetworkConfig`! I did not change it in TruffleConfig though, but because it was not a direct match.